src/main/docker/jre/Dockerfile

```diff
@@ -44,6 +44,10 @@ FROM stage1 AS stage2
 COPY install-ca-cert-java.sh /wd/startup/startup.d/
 RUN chmod 711 /wd/startup/startup.d/install-ca-cert-java.sh
 
+# Update java.security file
+COPY update-java-security-file.sh /wd/startup/startup.d/
```

Comment:
I don't understand why this is a startup script.  Isn't this just something we should be doing when building the image, i.e. before container startup?